### PR TITLE
Add Meeting endpoint

### PR DESCRIPTION
**Pull request by clerx**
Create meetings
Get meetings
Remove meetings
Attach attendees
Remove attendees

### CODE REVIEW

File: Meetings/Meeting.php [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+    /**
+     * @param $data
+     * @return Meeting
+     */
+    public static function initializeWithRawData($data)
+    {
+        $meeting = new Meeting();
+        foreach ($data as $key => $value) {
+            switch ($key) {
```
Comment (carakas) [inline Meetings/Meeting.php]: you only have a default, why do you use a switch case?

Comment (gclx) [reply Meetings/Meeting.php]: Copied it from Product.php tested it and it worked. But shouldn't be there 👍 

Removed in 5a1e46d9a38e2f22b76e11d07bf86eaaff39efe9

**Comment by clerx** [comment]
Added functionality to directly attach meetings to a company.

@carakas This is undocumented in the Teamleader API but is available for use (only in Teamleader API v1).

**Comment by clerx** [comment]
Added a function for the Teamleader getMeetings.php endpoint in 77556bb1c137a6a9d0f522291ef70a0cf842a877.

**Comment by carakas** [comment]
Released https://github.com/sumocoders/Teamleader/releases/tag/1.13.0

**Pull request merged**